### PR TITLE
Statistics script: print table as counts vs. percentages

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adds a command line argument to print the statistics as a count of each value type, rather than percentages.

### CODE REVIEW

File: scripts/statistics.js [hunk 1]
```diff
@@ -23,6 +23,12 @@ const { argv } = require('yargs').command(
         describe: 'Show statistics for all browsers within BCD',
         type: 'boolean',
         nargs: 0,
+      })
+      .option('asCount', {
```
Comment (caugner) [inline scripts/statistics.js:27]: Not opposed to this naming (being consistent with the function parameters below), but wondering whether an all-lowercase option would be more "conventional" (don't recall seeing mixed case options in the wild):

```suggestion
      .option('counts', {
```

**Review by caugner — COMMENTED** [review]
Hm, I ran `npm run stats -c` in this branch, but it didn't have the expected effect:

```console
% npm run stats -c                                                                                                                                  ✹

> @mdn/browser-compat-data@4.1.7 stats
> node scripts/statistics

Status as of version 4.1.7 (released on 2022-02-08) for web platform features: 

| browser | real values | ranged values | `true` values | `null` values |
| --- | --- | --- | --- | --- |
| total | 90.75% | 1.92% | 3.76% | 3.58% |
| chrome | 92.81% | 0.00% | 3.88% | 3.30% |
| chrome android | 91.08% | 0.00% | 5.62% | 3.29% |
| edge | 94.47% | 2.49% | 0.10% | 2.94% |
| firefox | 94.24% | 0.00% | 2.73% | 3.04% |
| ie | 91.82% | 0.37% | 4.56% | 3.25% |
| safari | 91.01% | 0.38% | 3.91% | 4.70% |
| safari ios | 90.63% | 0.60% | 4.06% | 4.71% |
| webview android | 79.90% | 11.54% | 5.18% | 3.38% |
```

**Review by caugner — CHANGES_REQUESTED** [review]
Doesn't seem to work as expected, see above.

**Comment by queengooborg** [comment]
> I ran `npm run stats -c` in this branch

You've actually got to run `npm run stats -- -c` instead, without the double dashes in front of arguments, they get passed to `npm` rather than the script it's running...  (That's my least favorite trait about `npm`...urgh.)

**Comment by caugner** [comment]
> You've actually got to run `npm run stats -- -c` instead,

Ouch, but thanks, works like a charme!

File: scripts/statistics.js [hunk 2]
```diff
@@ -162,14 +168,28 @@ const getStats = (folder, allBrowsers) => {
...
+
 /**
  * Print statistics of BCD
  *
  * @param {object.<string, VersionStats>} stats The stats object to print from
  * @param {string} folder The folder to show statistics for (or all folders if blank)
  * @returns {void}
  */
-const printStats = (stats, folder) => {
+const printStats = (stats, folder, counts) => {
```
Comment (caugner) [inline scripts/statistics.js:192]: Just noticed now that JSDoc is missing for this new parameter.

**Review by caugner — APPROVED** [review]
LGTM, noticed a last nit, but feel free to merge once resolved without requesting another review.

**Pull request merged**